### PR TITLE
Remove checked exception in RMMonitoring

### PR DESCRIPTION
**Pull request by fviale**

### CODE REVIEW

File: rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/util/RMListenerProxy.java [hunk 1]
```diff
@@ -145,8 +145,8 @@ private void checkCounter(RMEvent event) {
             logger.warn("Local event counter is " + counter + " vs. rm event counter " + event.getCounter());
             try {
                 this.target.getMonitoring().removeRMEventListener();
-            } catch (RMException e) {
-                logger.error(e.getMessage(), e);
+            } catch (Exception e) {
```
Comment (mcompastie) [inline rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/util/RMListenerProxy.java]: Can we be a bit more selective on the exception to be intercepted ?

File: rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/util/RMListenerProxy.java [hunk 2]
```diff
@@ -127,7 +127,7 @@ public boolean init(String url, Credentials credentials) throws RMException, Key
     public BooleanWrapper disconnect() {
         try {
             target.getMonitoring().removeRMEventListener();
-        } catch (RMException ignored) {
+        } catch (Exception ignored) {
```
Comment (mcompastie) [inline rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/util/RMListenerProxy.java]: Can we be a bit more selective on the exception to be intercepted ?

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/policy/RMAwareNodeSourcePolicy.java [hunk 3]
```diff
@@ -76,8 +76,8 @@ public void shutdown(Client initiator) {
         if (!rmShuttingDown) {
             try {
                 rmMonitoring.removeRMEventListener();
-            } catch (RMException e) {
-                logger.error("Error when removing event listener", e);
+            } catch (Exception e) {
```
Comment (mcompastie) [inline rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/policy/RMAwareNodeSourcePolicy.java]: Can we be a bit more selective on the exception to be intercepted ?

File: rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/util/RMListenerProxy.java (hunk 2 above)
Comment (fviale) [reply rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/util/RMListenerProxy.java]: not in the context of removal, close, clean, we want to be as large as possible

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/policy/RMAwareNodeSourcePolicy.java (hunk 3 above)
Comment (fviale) [reply rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/policy/RMAwareNodeSourcePolicy.java]: same answer

**Pull request merged**